File: starterkits/csharp/Bot.cs
using System;
using System.Collections.Generic;
using System.Linq;
using Blitz2021;
using static Blitz2021.GameCommand;
using static Blitz2021.GameCommand.UnitAction;
using static Blitz2021.Map;

namespace Blitz2020
{
    public class Bot
    {
        public static string NAME = "MyBot C#";

        public Bot()
        {
            // initialize some variables you will need throughout the game here
        }

        /*
        * Here is where the magic happens, for now the moves are random. I bet you can do better ;)
        *
        * No path finding is required, you can simply send a destination per unit and the game will move your unit towards
        * it in the next turns.
        */
        public GameCommand nextMove(GameMessage gameMessage)
        {
            Crew myCrew = gameMessage.getCrewsMapById[gameMessage.crewId];
            int mapSize = gameMessage.map.getMapSize();


            List<GameCommand.Action> actions = myCrew.units
                .Select(c => new UnitAction(UnitActionType.MOVE, c.id, this.getRandomPosition(mapSize)))
                .ToList<GameCommand.Action>();

            return new GameCommand(actions);
        }

        public Position getRandomPosition(int size)
        {
            Random rand = new Random();
            return new Position(rand.Next(size), rand.Next(size));
        }
    }
}

File: starterkits/python/bot.py
from typing import List
from game_message import GameMessage, Position, Crew
from game_command import Action, UnitAction, UnitActionType
import random


class Bot:

    def get_next_move(self, game_message: GameMessage) -> List[Action]:
        """
        Here is where the magic happens, for now the moves are random. I bet you can do better ;)

        No path finding is required, you can simply send a destination per unit and the game will move your unit towards
        it in the next turns.
        """
        my_crew: Crew = game_message.get_crews_by_id()[game_message.crewId]

        actions: List[UnitAction] = [UnitAction(UnitActionType.MOVE,
                                                unit.id,
                                                self.get_random_position(
                                                    game_message.map.get_map_size())) for unit in my_crew.units]

        return actions

    def get_random_position(self, map_size: int) -> Position:
        return Position(random.randint(0, map_size - 1), random.randint(0, map_size - 1))
